blogs/lokiless/lokiless.md

```diff
@@ -0,0 +1,282 @@
+# Deploying Network Observability without Loki
+
+This blog post talks about a new feature that comes with Network Observability (NetObserv) 1.4: the ability to deploy it without Loki, coupled with exporting flows to an alternative storage. We will try to explain what this new feature means to you, users, and to us, NetObserv developers.
```

Comment:
```suggestion
This blog post talks about a new feature that comes with Network Observability (NetObserv) 1.4: the ability to deploy it without Loki, coupled with exporting flows to an alternative storage. In this article, we will desmontrate a possible configuration to consume flows outside of Loki.
```

Reply:
I said that because I wrote a couple of things about why we did that, what's the pros and cons, and some thoughts about the possible follow-ups that this feature enables, etc.

Reply:
yeah the sentence can keep the "exploration" side of the article but without being too vague on the content. WDYT ?

Reply:
So I reformulated base on that, but trying to convey all that I want:

> It clarifies what this change is and what it isn't. Then it demonstrates the optional configuration for consuming flows outside of Loki. Finally, it explores potential avenues for future development that this feature enables.